### ISSUE TITLE
Carousel don't recognize the data-interval attribute

### ISSUE THREAD

**Issue by speller** [creativetimofficial/material-kit#104]
1. Implement the carousel example from the docs: https://demos.creative-tim.com/material-kit/docs/2.0/components/carousel.html
2. Add the `data-interval="10000"` attribute to the carousel tag.

What is expected:
Slide interval must change.

What happen instead:
Slide interval not changed and remain default. Even setting `data-interval="false"` has no effect.

**Comment by dragosct10** [comment]
Hi, @speller! Where did you add this attribute on which div from the carousel? 

Regards, Dragos

**Comment by speller** [comment]
@dragosct10 

Here is my carousel code:

```
<div id="carousel50" class="carousel slide" data-ride="carousel" data-interval="100000">
  <ol class="carousel-indicators">
    <li data-target="#carousel50" data-slide-to="0" class="active"></li>
    <li data-target="#carousel50" data-slide-to="1"></li>
    <li data-target="#carousel50" data-slide-to="2"></li>
  </ol>
  <div class="carousel-inner" data-interval="false">
    <div class="carousel-item active">
      <img class="d-block w-100" src="/static/tmp/prop1.jpg"/>
    </div>
    <div class="carousel-item">
      <img class="d-block w-100" src="/static/tmp/prop2.jpg"/>
    </div>
    <div class="carousel-item">
      <img class="d-block w-100" src="/static/tmp/prop3.jpg"/>
    </div>
  </div>
  <a class="carousel-control-prev" href="#carousel50" role="button" data-slide="prev">
    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
    <span class="sr-only">Previous</span>
  </a>
  <a class="carousel-control-next" href="#carousel50" role="button" data-slide="next">
    <span class="carousel-control-next-icon" aria-hidden="true"></span>
    <span class="sr-only">Next</span>
  </a>
</div>

```

**Comment by dragosct10** [comment]
Hi! @speller! Sorry for the late response. The data-interval it's not working because already in the project in `assets/js/material-kit.js` if you will search for carousel you will see that interval is set on 3000, please go there and put the interval or remove the interval from the `material-kit.js` file and put the attribute on the div .carousel

**Comment by speller** [comment]
@dragosct10 
Your workaround works, thanks. But what do you plan to fix this issue and avoid hard-coded interval values?

**Comment by dragosct10** [comment]
Hi, @speller! Yes, we will fix this in the next update of course, because we want to avoid situations like this. 

Regards, 
Dragos

**Comment by groovemen** [comment]
Hello @speller,

Thank you for your interest in working with our products, this issue is solved in the latest version (v2.0.4).

All the best,
Stefan
